lib/identity_cache/query_api.rb

```diff
@@ -227,7 +227,7 @@ def resolve_cache_miss(id)
         record = self.includes(cache_fetch_includes).reorder(nil).where(primary_key => id).first
```

Comment:
Talked with @Sirupsen about this in person and we plan on banning embedded an association between models that use different databases at the application level.